app/controllers/application_controller.rb

```diff
@@ -2,11 +2,19 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  
```

Comment:
Atom puts 1 empty line on the end of the file. I may have started working down too far. I try to keep an eye on weird things like this, but with a decently sized project, it's hard not to overlook something.